sql_user_check_mysql/README.md

```diff
@@ -0,0 +1,19 @@
+# MYSQL User Check
+
+database_stats.sh script can be used to test connectivity with MYSQL database as well as fetch some stats and metadata. 
+The script supports 2 modes which can be specified with -m flag while running the script. One mode is stats mode which 
+will just print info like list of databases with their size. Count of tables, rows, column, size per schema. 
+Distribution of data by data type. Stats mode is the default mode. Another mode is full_metadata which will print 
+complete information of all columns like their data type, constraints etc.
+
+From `sql_user_check_mysql` directory, run these commands depending on mode of script. 
+
+stats mode:
+```shell
+./database_stats.sh -u <DATABASE_USER> -h <DATABASE_HOSTNAME> -o <OUTPUT_FILE> -P <PASSWORD>
```

Comment:
Don't ask for password in plain text since it will get stored in shell history. Use https://stackoverflow.com/questions/4316730/hiding-user-input-on-terminal-in-linux-script solution to ask for password

Reply:
now prompting user for a password.